site/en/install/pip.md

```diff
@@ -476,7 +476,7 @@ The following NVIDIA® software are only required for GPU support.
     installation to be sure you're running the latest version.
 
     ```bash
-    pip install --upgrade pip
+    python -m pip install --upgrade pip
```

Comment:
There are two commands a few paragraphs down, in this "windows native" tab that use `python3` instead of `python`. Can you updater those too?

Reply:
@BLUE-DEVIL1134 PTAL